tests/kafkatest/services/performance/producer_performance.py

```diff
@@ -62,6 +63,7 @@ def parse_stats(line):
             }
         last = None
         for line in node.account.ssh_capture(cmd):
+            self.maybe_start_jmx_tool(idx, node)
```

Comment:
@lindong28 Thanks for this observation. I added a fix in https://github.com/confluentinc/ducktape/pull/107

The fix is now available from pypi as version 0.3.5 - you'll want to update the ducktape dependency in tests/setup.py

Note this is actually asynchronous even without running command as a background process with "&", so you should be able to run:

```
producer_output = node.account.ssh_capture(cmd)
wait_until(producer is alive)
self.start_jmx_tool(node)

for line in producer_output:
  process(line)
```